packages/react/src/components/AccountSettings/ConfigureTOTP/ConfigureTOTP.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import QRCode from 'qrcode';
+
+import {
+  setupTOTP,
```

Comment:
nit: alphabetize

Reply:
Done in [6fee5d1](https://github.com/aws-amplify/amplify-ui/pull/2969/commits/6fee5d15f7c6f6bdbdf9e1d572cb4c91bd6f4700)!